style.css

```diff
@@ -430,3 +429,161 @@ div.border-bootom {
   color: #fff;
   background: #2230d2;
 }
+
+.navbar .links {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    margin: 10px 20px;
+    padding: 0 60px;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navbar .links {
+    display: flex;
+  }
+
+  ul.links {
+    margin-top: 0;
+  }
+
+  .main-section {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    padding: 50px 130px;
+  }
```

Comment:
- Good job on the work done so far, but there are some differences between your project and the chosen design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project(Please pay attention to the marked section in the Figma template attached below):

- This section is missing background images as seen in the Figma template, please fix this 🙏. To fix this issue, I advise you consider removing the `background-position: center` property used in the mobile version 
- The bottom left part of this section has a border-radius which is absent in your design, please fix this 😄 
<img width="947" alt="tntenwrynw" src="https://user-images.githubusercontent.com/73607512/207826478-3ac50af8-b522-4c32-992e-8b4039ce71be.PNG">

Figma template:
<img width="574" alt="t44ttt" src="https://user-images.githubusercontent.com/73607512/207827056-cfb77082-66eb-46d8-bf29-781e350428cf.PNG">



The See Resume button is not styled as it is in the given Figma template, please fix this 🙏

The top right of the section has a border-radius which is absent in your design, please fix this 👍

Please fix all the angled arrows for the technologies and make sure they are the same as the given design template 👍 👍

Reply:
Fixed.